AppCreationScripts/apps.json

```diff
@@ -1,44 +0,0 @@
-{
-    "Sample": {
-    "Title": "Calling Microsoft Graph from a Windows Universal Platform App (UWP)",
-    "Level": 200,
-    "Client": "UWP, MSAL.NET"
-  },
-  "AppRegistrations": [
-    {
-      "x-ms-id": "active-directory-dotnet-native-uwp-v2",
-      "x-ms-name": "active-directory-dotnet-native-uwp-v2",
-      "x-ms-version": "2.0",
-      "replyUrlsWithType": [
-        {
-          "url": "https://login.microsoftonline.com/common/oauth2/nativeclient",
```

Comment:
we need to work with Manoj to have the portal compute the redirect URI of the solution it provides as a zip file.